### PR TITLE
Improve park cheats tab layout

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
As proposed and discussed in #13160, I have improved the layout of the park cheats tab.

* The 'general' group has been split in two, introducing a new 'objective' group.
* The 'park parameters' button has been removed, as 'Scenario Options' is already accessible via the cheats menu.
* 'Force weather' has been relabelled 'Change weather to'.
* 'Freeze weather' is now a checkbox instead of a button.
* The 'staff' group has been split in two, introducing a new 'maintenance' group.

### Discussion points

* [x] Should we shrink the park rating spinner, so there is a little more wiggle room for translations?
* [x] Should we consider relabelling 'Freeze weather' with 'Disable weather changes'?
* [x] Should we consider relabelling 'Mowed grass' with 'Mow grass' — make it clear that it's an action, like 'Clear grass'?
* [x] In the proposal, 'Water plants' and 'Plants do not age' were on the same line, but this apparently causes some localisation problems. Deferred.

### Before/after (English)

cheats-old-english cheats-new-english

### Before/after (French)

cheats-old-french cheats-new-french

### CODE REVIEW

File: data/language/en-GB.txt [hunk 1]
```diff
@@ -1967,7 +1967,7 @@ STR_2763    :???
 STR_2765    :Large Tram
 STR_2766    :Win scenario
 STR_2767    :Freeze weather
-STR_2768    :Unfreeze weather
+STR_2768    :<not used anymore>
```
Comment (Gymnasiast) [inline data/language/en-GB.txt:1970]: We used to do this, but for the last 1-2 years we have just removed them. (Strings don't get reused anyway and leaving them out makes it easier to detect old strings in the translations.)

**Comment by Gymnasiast** [comment]
In the screenshot, "Staff" and "Maintenance" both have the same name. Has this since been fixed in the code?

File: src/openrct2/localisation/StringIds.h [hunk 2]
```diff
@@ -2078,7 +2078,7 @@ enum
     STR_CHEAT_LARGE_TRAM_GUESTS = 2765,
     STR_CHEAT_WIN_SCENARIO = 2766,
     STR_CHEAT_FREEZE_WEATHER = 2767,
-    STR_CHEAT_UNFREEZE_WEATHER = 2768,
+    // STR_CHEAT_UNFREEZE_WEATHER = 2768,
```
Comment (Gymnasiast) [inline src/openrct2/localisation/StringIds.h:2081]: Remove this line entirely.

File: data/language/en-GB.txt (hunk 1 above)
Comment (AaronVanGeffen) [reply data/language/en-GB.txt:1970]: Ah, I found one more such entry in the file -- that's why I went with this format. I'll remove the lot tomorrow, then.

**Review by tupaschoal — COMMENTED** [review]
> The 'staff' group has been split in two, introducing a new 'maintenance' group.

I found it a bit weird to have two groups with the same title in the same window

> Should we shrink the park rating spinner, so there is a little more wiggle room for translations?

I'd say yes, but it wouldn't help portuguese by a lot, so I guess if you like it being aligned with all other similar elements in the window, then no.

> Should we consider relabelling 'Freeze weather' with 'Disable weather changes'?

Yes!

> Should we consider relabelling 'Mowed grass' with 'Mow grass' — make it clear that it's an action, like 'Clear grass'?

Yes!

**Comment by AaronVanGeffen** [comment]
I've updated the strings in question, and the screenshots in the OP to reflect the changes.

**Comment by Gymnasiast** [comment]
> Should we shrink the park rating spinner, so there is a little more wiggle room for translations?

It would probably be better.

> Should we consider relabelling 'Freeze weather' with 'Disable weather changes'?

Good idea. It's clearer.

> Should we consider relabelling 'Mowed grass' with 'Mow grass' — make it clear that it's an action, like 'Clear grass'?

Yes, it's clearer and more consistent.

> In the proposal, 'Water plants' and 'Plants do not age' were on the same line, but this apparently causes some localisation problems. Deferred.

Having them on separate lines is best for localisation, I think.

**Comment by Umdlye** [comment]
Unless there's a good reason for keeping them this way, I would love to see "Park" and "Ducks" in the cheat buttons written in lowercase for consistency.

**Comment by AaronVanGeffen** [comment]
I believe that's everything for which consensus has been reached. The screenshots in the OP have been updated to reflect the latest changes.

Unless there are urgent matters, I would prefer to leave further improvements to another PR. This is done as far as I'm concerned.

**Review by Broxzier — APPROVED** [review]

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**